s32/drivers/s32ze/Eth_NETC/src/Netc_Eth_Ip_Irq.c

```diff
@@ -108,7 +108,7 @@ static inline void Netc_Eth_Ip_MSIX_Tx(uint8 CtrlIndex);
  *
  * @param CtrlIndex
  */
-static inline void Netc_Eth_Ip_MSIX_Rx(uint8 CtrlIndex);
+void Netc_Eth_Ip_MSIX_Rx(uint8 CtrlIndex);
```

Comment:
can we avoid this and use `Netc_Eth_Ip_x_MSIX_RxEvent` instead?

Reply:
we can not use `Netc_Eth_Ip_x_MSIX_RxEvent` when using `DT_INST` for NETC. Because `x` number in `Netc_Eth_Ip_x_MSIX_RxEvent` is the Station Interface (SI) instance index, it not align with the devicetree instance index.